<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayerWindow</class>
 <widget class="QWidget" name="MusicPlayerWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>400</width><height>500</height></rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
      background-image: url("Images/Backgrounds/1.jpeg");
      background-repeat: no-repeat;
      background-position: center;
    }
   </string>
  </property>

  <!-- Menu Button -->
  <widget class="QPushButton" name="menuButton">
   <property name="geometry"><rect><x>10</x><y>10</y><width>40</width><height>40</height></rect></property>
   <property name="text"><string>≡</string></property>
   <property name="styleSheet">
    <string notr="true">
     QPushButton {
       background-color: #3c2c5e;
       border-radius: 10px;
       color: white;
       font-size: 18px;
     }
    </string>
   </property>
  </widget>

  <!-- Song Title Widget -->
  <widget class="QLabel" name="songTitle">
   <property name="geometry"><rect><x>100</x><y>15</y><width>200</width><height>30</height></rect></property>
   <property name="text"><string>Favorite Song</string></property>
   <property name="alignment"><set>Qt::AlignCenter</set></property>
   <property name="styleSheet">
    <string notr="true">
     QLabel {
              font-family: "Orbitron";
              font-size: 18pt;
              font-weight: bold;
              color: rgba(191, 64, 191, 200);  /* Neon magenta */

     }
    </string>
   </property>
  </widget>

  <!-- Album Cover -->
  <widget class="QLabel" name="albumCover">
   <property name="geometry"><rect><x>100</x><y>60</y><width>200</width><height>200</height></rect></property>
   <property name="pixmap"><pixmap>:/resourses/Images/Covers/track1.jpg</pixmap></property>
   <property name="scaledContents"><bool>true</bool></property>
   <property name="styleSheet">
    <string notr="true">
     QLabel {
       border-radius: 80px;
     }
    </string>
   </property>
  </widget>

  <!-- Progress Bar -->
  <widget class="QSlider" name="progressBar">
   <property name="geometry"><rect><x>50</x><y>270</y><width>300</width><height>20</height></rect></property>
   <property name="orientation"><enum>Qt::Horizontal</enum></property>
   <property name="styleSheet">
    <string notr="true">
     QSlider::groove:horizontal {
       background: #a86cc1;
       height: 6px;
       border-radius: 3px;
     }
     QSlider::handle:horizontal {
       background: white;
       border: 1px solid #a86cc1;
       width: 12px;
       margin: -3px 0;
       border-radius: 6px;
     }
    </string>
   </property>
  </widget>

  <!-- Time Labels -->
  <widget class="QLabel" name="timeStart">
   <property name="geometry"><rect><x>50</x><y>295</y><width>50</width><height>20</height></rect></property>
   <property name="text"><string>1:02</string></property>
   <property name="styleSheet"><string notr="true">color: white;</string></property>
  </widget>
  <widget class="QLabel" name="timeEnd">
   <property name="geometry"><rect><x>300</x><y>295</y><width>50</width><height>20</height></rect></property>
   <property name="text"><string>2:43</string></property>
   <property name="styleSheet"><string notr="true">color: white;</string></property>
  </widget>

  <!-- Control Button Container -->
  <widget class="QWidget" name="buttonContainer">
   <property name="geometry"><rect><x>80</x><y>370</y><width>240</width><height>60</height></rect></property>
   <property name="styleSheet">
    <string notr="true">
     QWidget {
       background-color: rgba(75 , 46 , 111 , 178);
       border-radius: 15px;
     }
    </string>
   </property>

   <!-- Previous Button -->
   <widget class="QPushButton" name="prevButton">
    <property name="geometry"><rect><x>10</x><y>10</y><width>40</width><height>40</height></rect></property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        color: white;
        font-size: 20px;
      }
     </string>
    </property>
   </widget>

   <!-- Pause Button -->
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry"><rect><x>90</x><y>5</y><width>60</width><height>50</height></rect></property>
    <property name="text"><string>⏸</string></property>
    <property name="icon">
      <iconset>
        <normaloff>:resourses/Images/Icons/pause_button_icon.png</normaloff>
        <normalon>:resourses/Images/Icons/pause_button_icon.png</normalon>
      </iconset>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #a86cc1;
        border-radius: 25px;
        color: white;
        font-size: 20px;
      }
     </string>
    </property>
   </widget>

   <!-- Next Button -->
   <widget class="QPushButton" name="nextButton">
    <property name="geometry"><rect><x>190</x><y>10</y><width>40</width><height>40</height></rect></property>
    <property name="text"><string>⏩</string></property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: ;
        color: white;
        font-size: 18px;
      }
     </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>